core/merge.js

```diff
@@ -792,6 +792,24 @@ const needsFileidMigration = (
   fileid /*: ?string */
 ) /*: boolean %checks */ => existing.fileid == null && fileid != null
 
+const sideNames = new Set(['local', 'remote'])
+
+const detectSingleSide = doc => {
+  if (doc.sides) {
+    for (const sideName of sideNames) {
+      if (doc.sides[sideName] && !doc.sides[otherSide(sideName)]) {
+        return sideName
+      }
+    }
+  }
+}
+
+const convertToDestinationAddition = (side, dst, src) => {
```

Comment:
I'll think about it. Maybe do both?

Reply:
Doing both may look strange since move default export is already a function with another function as property:

```js
move()
move.child()
move.convertMoveToDestinationAddition() // move x2
move.convertToDestinationAddition() // better?
```

Reply:
ah right. I thought `move()` was exported as a property.
The second one looks good to me.